<robot name="obj_01">
  <handle name="lateral_top" clearance="0.035">
    <position xyz="0 0 0.012" wxyz="1 0 0 0"/>
    <link name="base_link"/>
    <mask>1 1 1 1 1 0</mask>
  </handle>
  <handle name="lateral_bottom" clearance="0.035">
    <position xyz="0 0 -0.025" wxyz="0 1 0 0"/>
    <link name="base_link"/>
    <mask>1 1 1 1 1 0</mask>
  </handle>
  <handle name="top" clearance="0.02">
    <position xyz="0 0 0.025" xyzw="0 0.7071067811865476 0 0.7071067811865476"/>
    <link name="base_link"/>
    <mask>1 1 1 0 1 1</mask>
  </handle>
  <handle name="bottom" clearance="0.02">
    <position xyz="0 0 -0.025" xyzw="0 -0.7071067811865476 0 0.7071067811865476"
              />
    <link name="base_link"/>
    <mask>1 1 1 0 1 1</mask>
  </handle>
</robot>
